scripts/constants.mjs

```diff
@@ -10,6 +10,7 @@ export const CKEDITOR5_ROOT_PATH = upath.join( import.meta.dirname, '..' );
 export const CKEDITOR5_PACKAGES_PATH = upath.join( CKEDITOR5_ROOT_PATH, PACKAGES_DIRECTORY );
 export const CKEDITOR5_COMMERCIAL_PATH = upath.join( CKEDITOR5_ROOT_PATH, '..', '..' );
 export const CKEDITOR5_COMMERCIAL_PACKAGES_PATH = upath.join( CKEDITOR5_COMMERCIAL_PATH, PACKAGES_DIRECTORY );
+export const IS_STANDALONE_REPOSITORY = upath.basename( upath.join( CKEDITOR5_ROOT_PATH, '..' ) ) !== 'external';
```

Comment:
```suggestion
export const IS_ISOLATED_REPOSITORY = upath.basename( upath.join( CKEDITOR5_ROOT_PATH, '..' ) ) !== 'external';
```

Reply:
Renamed in [a5355f9](https://github.com/ckeditor/ckeditor5/pull/19173/commits/a5355f94d7c7d5805986a4bda94648403ecce07d).